variables.tf

```diff
@@ -26,3 +26,41 @@ variable "di_connectivity_tester_web_ip" {
   description = "Mock EC2 web instance."
   default     = "10.1.6.21"
 }
+
+variable "instance_type" {
+  default     = "t2.micro"
+  description = "EC2 instance size unless specified."
+}
+
+variable "key_name" {
+  default     = "test_instance"
+  description = "Key name for login."
+}
+
+variable "dp_web_private_ip" {
+  default     = "10.1.6.100"
+  description = "Web server address"
+}
+
+variable "bucket_src_path" {
+  description = "Source bucket object path"
+  default     = ""
+}
+
+variable "local_dest_path" {
+  description = "Local instance destination full file path"
+  default     = "C:\tmp"
+}
+
+variable "TCPPorts" {
```

Comment:
isn't this defined by the applications defined in this? tf, not sure why the parent would dictate this?
same applies for `asg_min` and `asg_max`